Exercices/JS06/index.js

```diff
@@ -0,0 +1,11 @@
+"use strict"; 
+alert("Enter the current hour");
+let hour = prompt("insert hour between 0 - 23:");
+Number(hour);
+if (hour < 0 || hour > 23) {
+    alert("Input incorrect, please insert hour between 0 - 23");
+} else if(hour < 8 || hour >= 18){
+    alert("We are closed!");
+} else if(hour >= 8 && hour < 18){
```

Comment:
If you don't expect any other cases, it's last condition, you should use `else {}` instead of `else if {}`.